build/testdata/024.golden

```diff
@@ -4,8 +4,8 @@
 str.rewrite(".", "/")
 
 str.rewrite(
-    before = ".",
     after = "/",
+    before = ".",
```

Comment:
I changed it so that buildifier will sort keyword arguments on calls that are accessed via dot expressions that start with a literal expression. str.rewrite is such a call